CRM/CiviAwards/BAO/AwardDetail.php

```diff
@@ -119,4 +120,28 @@ private static function validateReviewFields(array $params) {
     $reviewFieldsValidator->validate($reviewFields);
   }
 
+  /**
+   * Validates award_manager options to ensure legal options are passed.
+   *
+   * @param array $params
+   *   Request Parameters.
+   *
+   * @throws \Exception
+   */
+  private static function validateAwardManager(array $params) {
+    if (empty($params['award_manager'])) {
+      throw new Exception('Award Manager should not be empty');
+    }
```

Comment:
Let's move the code above to the proposed validatemandatory method. The logic below can be left since it validates valid award managers/contacts.